core/src/main/java/hudson/model/listeners/ItemListener.java

```diff
@@ -171,23 +171,8 @@ public void register() {
         return ExtensionList.lookup(ItemListener.class);
     }
 
-    // TODO JENKINS-21224 generalize this to a method perhaps in ExtensionList and use consistently from all listeners
-    private static void forAll(final Consumer<ItemListener> consumer) {
-        for (ItemListener l : all()) {
-            try {
-                consumer.accept(l);
-            } catch (RuntimeException x) {
-                LOGGER.log(Level.WARNING, "failed to send event to listener of " + l.getClass(), x);
-            }
-        }
-    }
-
     public static void fireOnCopied(final Item src, final Item result) {
-        forAll(l -> {
-            if (l != null) {
```

Comment:
Deleting null check introduced in #4458 for no reason I can follow. `ExtensionList.lookup` should never return null entries.